src/instructions.spec.ts

```diff
@@ -469,19 +469,18 @@ describe('Cortex-M0+ Instruction Set', () => {
     expect(registers.V).toEqual(false);
   });
 
-  it('should execute an `cmp	r2, r0` instruction', async () => {
+  it('should correctly unset carry when executing an `cmp	r2, r0` instruction', async () => {
     await cpu.setPC(0x20000000);
     await cpu.writeUint16(0x20000000, opcodeCMPregT1(r2, r0));
-    await cpu.setRegisters({ r0: 0xb71b0000 });
-    await cpu.setRegisters({ r2: 0x00b71b00 });
-    await cpu.setRegisters({ xPSR: 0x61000000 });
+    await cpu.setRegisters({ r0: 0xb71b0000, r2: 0x00b71b00 });
```

Comment:
Didn't you also want to set C/Z to true?

Reply:
I tested it now, the previous flag can be any value so I removed the set.

Reply:
okay great!